python/ray/experimental/dag/tests/test_plot.py

```diff
@@ -0,0 +1,30 @@
+import os
+import ray
+
+
+def test_basic_plot(shared_ray_instance):
+    @ray.remote
+    def a(*args, **kwargs):
+        pass
+
+    tmp1 = a.options(name="tmp1").bind()
```

Comment:
Could you also write some tests without name? It should work out of the box.

We can use the pkg.func as the name.